make.conf

```diff
@@ -67,6 +67,9 @@ OVMF_DIR ?= $(shell for d in $(OVMF_SEARCH); do [ -d "$$d" ] && break; done; ech
 OVMF_CODE = $(OVMF_DIR)/OVMF_CODE.secboot.fd
 OVMF_VARS = $(OVMF_DIR)/OVMF_VARS.fd
 
+NOGUI = -nodefaults \
+	-device VGA -vnc :9000 -serial mon:stdio
```

Comment:
Should we use regular vnc ports:  ```-vnc :1  ``` or something

Reply:
Probably.  And I guess I need a copy of my next-unused-vnc-port script in here.

Reply:
Yeah, I'd switch NOGUI name to VNC and move the -serial mon:stdio to the QEMU_CMD_CORE